cypress/e2e/communityTemplates.test.ts

```diff
@@ -211,6 +211,8 @@ describe('Community Templates', () => {
     it.skip('takes you to github readme when you click on the Community Templates button', () => {
       cy.getByTestID('community-template-readme-overlay-button').click()
       cy.get('.markdown-format').should('contain', 'Setup Instructions')
+
+      //we should confirm the link works
```

Comment:
what's this? is this a legit todo?

Reply:
should have added a better note, but yes its to check the link works and leads to the proper template.md file. Though we have this as a unit test, i guess we should do it in the e2e possibly